bin/einhorn

```diff
@@ -318,6 +318,10 @@ if true # $0 == __FILE__
       Einhorn::State.config[:max_upgrade_additional] = Integer(n)
     end
 
+    opts.on('--gc-before-fork', 'Run the GC three times before forking to improve memory sharing for copy-on-write.') do
+      Einhorn::State.config[:gc_for_cow_before_fork] = true
```

Comment:
Sorry for the nit, but would you mind if we make these two names of the same. e.g., `--gc-before-fork` and `[:gc_before_fork]`.

Reply:
Sure, done